package.yaml

```
@@ -1,5 +1,5 @@
 name: ratel-wai
-version: 1.0.5
+version: 1.0.6
```

This needs to bump the minor version number to communicate that something was added. 

```suggestion
version: 1.1.0
```